OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -847,6 +838,24 @@ public void ResolveOrder(Actor self, Order order)
 			}
 		}
 
+		void DoMove(Target target, bool queued)
+		{
+			var cell = self.World.Map.Clamp(this.self.World.Map.CellContaining(target.CenterPosition));
```

Comment:
omit `this`

Reply:
No longer relevant in the current version.